.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode
+.idea
```

Comment:
While `.idea` folder is really very local to the machine, `.idea` is a good folder to share with other collaborators. You can see that big projects as [Jest](https://github.com/facebook/jest) are sharing it.

Reply:
I'm not using IntelliJ as my main IDE, because my computer spec is too weak for it. I think this file is when I show my code to a friend in work because I have some TS issue. My local IDE setting for VSC or Jetbrains are default. I think it's best to keep it in gitignore file as it.